pallets/creator-staking/src/types/mod.rs

```diff
@@ -123,13 +127,13 @@ pub struct StakesInfo<
     pub(super) stakes: BoundedVec<EraStake<Balance>, MaxEraStakeValues>,
 }
 
-/// Represents an balance amount undergoing the unbonding process.
+/// Represents a balance amount that is currently unbonding.
 /// Since unbonding takes time, it's important to keep track of when and how much was unbonded.
 #[derive(
 Clone, Copy, PartialEq, Encode, Decode, Default, RuntimeDebug, TypeInfo, MaxEncodedLen,
 )]
 pub struct UnlockingChunk<Balance: MaxEncodedLen> {
-    /// Amount being unlocked
+    /// Amount being unbonded
```

Comment:
Unlocking Chunk, but "Amount being unbonded"?

Reply:
should be UnbondingChunk then?

Reply:
Not sure what is better, but need to use one word for all 100%